packages/grafana-ui/src/components/GraphNG/types.ts

```diff
@@ -18,3 +18,11 @@ export interface GraphNGLegendEvent {
   fieldIndex: DataFrameFieldIndex;
   mode: GraphNGLegendEventMode;
 }
+
+/**
+ * @alpha
+ */
+export interface StackingOptions {
```

Comment:
Lets reuse the config from:
https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/BarChart/types.ts#L8

Reply:
done!